Lottery.java

```diff
@@ -0,0 +1,13 @@
+import java.util.Random;
+
+public class Lottery {
+   private static final int LOTTERY_MAX_COUNT = 100;
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        int number = new Random().nextInt(LOTTERY_MAX_COUNT);
+        ball.setNumber(number);
+        ColorSupplier supplier = new ColorSupplier();
+        ball.setColor(String.valueOf(supplier.getRandomColor()));
```

Comment:
Please, do not use `String.valueOf` with ENUM values 
Use name() for getting String representation of enum constants
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants

Reply:
Uliana, thank You very much for improving my Java's skills ! :)